qiita_db/processing_job.py

```diff
@@ -174,7 +241,7 @@ def status(self):
 
         Returns
         -------
-        {'queued', 'running', 'success', 'error'}
+        {'queued', 'running', 'success', 'error', 'in_construction', 'waiting'}
```

Comment:
Minor, but the numpy way to do this would be:

```python
str
    The current status of the job, one of .....
```
Or something along those lines right?